packages/forms/src/UIForm/fieldsets/Fieldset/Fieldset.component.test.js

```diff
@@ -29,6 +29,44 @@ describe('Fieldset widget', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should not render fieldset with empty items', () => {
```

Comment:
Sure about this test? You have items in your schema here (so you have `fieldset` in the snapshot)